### ISSUE TITLE
Minimum required permissions for using schemaspy with different DBs

### ISSUE THREAD

**Issue by mor-lehr-deel** [schemaspy/schemaspy#1522]
Would be nice if the documentation states the minimum required privileges the DB User needs to have in every type of DB to successfully crawl it.
I.e it would be good to know that it doesn't need to access the tables themselves, but only the metadata, so for example for postgres it only needs access to pg_catalog and/or information_schema etc. (I didn't check if that's true or not yet :) ).
This is following the approach of monitoring tools like Zabbix or Datadog which explains how to create a dedicated user with the specific required permissions.

**Comment by hS6ZQZHwgT** [comment]
Yes, please, this would be great information to have!  We just reviewed the documentation and did not see this information.  We have multiple DB systems and definitely do not look forward to self-discovering the principles of least required access for each DBMS.
